src/site/content/en/react/how-amp-can-guarantee-fastness-in-your-nextjs-app/index.md

```diff
@@ -0,0 +1,99 @@
+---
+layout: post
+title: How AMP can guarantee fastness in your Next.js app
+authors:
+  - houssein
+subhead: |
+  Learn about the benefits and tradeoffs of supporting AMP in your Next.js app
+date: 2019-10-17
+draft: true
+codelabs: how-to-use-amp-in-nextjs
+---
+
+[AMP](https://amp.dev) (Accelerated Mobile Pages) is a web component framework that guarantees
```

Comment:
AMP is no longer "Accelerated Mobile Pages", it is just "AMP". Its used on desktop too so the name changed.